actors/abi/bigint/bigint.go

```diff
@@ -0,0 +1,250 @@
+package big
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"math/big"
+
+	cbor "github.com/ipfs/go-ipld-cbor"
+	"github.com/polydawn/refmt/obj/atlas"
+
+	cbg "github.com/whyrusleeping/cbor-gen"
+	"golang.org/x/xerrors"
+)
+
+const BigIntMaxSerializedLen = 128 // is this big enough? or too big?
+
+func init() {
+	cbor.RegisterCborType(atlas.BuildEntry(Int{}).Transform().
+		TransformMarshal(atlas.MakeMarshalTransformFunc(
+			func(i Int) ([]byte, error) {
+				return i.cborBytes(), nil
+			})).
+		TransformUnmarshal(atlas.MakeUnmarshalTransformFunc(
+			func(x []byte) (Int, error) {
+				return fromCborBytes(x)
+			})).
+		Complete())
+}
+
+type Int struct {
+	*big.Int
+}
+
+func NewInt(i int64) Int {
+	return Int{big.NewInt(0).SetInt64(i)}
+}
+
+func BigFromBytes(b []byte) Int {
+	i := big.NewInt(0).SetBytes(b)
+	return Int{i}
+}
+
+func BigFromString(s string) (Int, error) {
+	v, ok := big.NewInt(0).SetString(s, 10)
+	if !ok {
+		return Int{}, fmt.Errorf("failed to parse string as a big int")
+	}
+
+	return Int{v}, nil
+}
+
+func BigMul(a, b Int) Int {
+	return Int{big.NewInt(0).Mul(a.Int, b.Int)}
+}
+
+func BigDiv(a, b Int) Int {
+	return Int{big.NewInt(0).Div(a.Int, b.Int)}
+}
+
+func BigMod(a, b Int) Int {
+	return Int{big.NewInt(0).Mod(a.Int, b.Int)}
+}
+
+func BigAdd(a, b Int) Int {
+	return Int{big.NewInt(0).Add(a.Int, b.Int)}
+}
+
+func BigSub(a, b Int) Int {
+	return Int{big.NewInt(0).Sub(a.Int, b.Int)}
+}
+
+func BigCmp(a, b Int) int {
+	return a.Int.Cmp(b.Int)
+}
+
+func (bi Int) Nil() bool {
+	return bi.Int == nil
+}
```

Comment:
Remove this, it's unused and we should strive to make it as impossible as performant Go allows.